core/domain/email_manager.py

```diff
@@ -607,6 +607,23 @@ def send_feedback_message_email(recipient_id, feedback_messages):
         email_subject, email_body, feconf.NOREPLY_EMAIL_ADDRESS)
 
 
+def _can_receive_suggestion_email(user_id, exploration_id):
+    """Returns if user can receive suggestion email
+
+    Args:
+        recipient_id: str. ID of person that should receive the email.
+        exploration_id: str. ID of exploration that received new message.
+
+    Returns:
+        bool. True if user can receive the email, False otherwise."""
+    user_global_prefs = user_services.get_email_preferences(user_id)
+    user_exploration_prefs = (
+        user_services.get_email_preferences_for_exploration(
+            user_id, exploration_id))
+    return (user_global_prefs['can_receive_feedback_message_email']
```

Comment:
I'm starting to get worried about using a dict for this and user_exploration_prefs -- reading this code makes me worry about things like KeyErrors.

Can we introduce a UserGlobalPrefs domain object in core/domain/user_domain.py and ditto for UserExplorationPrefs, then pass these domain objects around instead? That's a better pattern, and more consistent with what we do elsewhere in the codebase.